power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/linear_pf_solver.hpp

```diff
@@ -42,6 +42,8 @@ if there are sources
 
 namespace power_grid_model {
 
+namespace math_solver::linear_pf {
```

Comment:
this should be

```suggestion
namespace power_grid_model::math_solver {

namespace linear_pf {
```

please also apply to all other files in this directory